sci-libs/vxl/ChangeLog

```diff
@@ -0,0 +1,8 @@
+# ChangeLog for sci-libs/vxl
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+*vxl-9999 (25 May 2015)
+
+  25 May 2015;  <chymera@gentoo.org> +metadata.xml, +vxl-9999.ebuild:
```

Comment:
you mean my email? yes, seriously, I have no idea where this is coming from and how to fix it. You probably remember that I tried to (with help from here and from the IRC) repeatedly :-/